src/plugins/saved_objects_management/public/management_section/objects_table/components/duplicate_result_flyout.tsx

```diff
@@ -65,15 +79,91 @@ const getErrorMessage = ({ error }: SavedObjectsImportError) => {
     return error.message;
   } else if (error.type === 'unsupported_type') {
     return unsupportedTypeErrorMessage;
+  } else if (error.type === 'missing_data_source') {
+    return i18n.translate('savedObjectsManagement.objectsTable.copyResult.missingDataSourceError', {
+      defaultMessage: 'Missing data source: {ds}',
+      values: { ds: error.dataSource },
+    });
+  } else if (error.type === 'missing_references') {
+    return i18n.translate('savedObjectsManagement.objectsTable.copyResult.missingReferencesError', {
+      defaultMessage: 'Missing references.',
+    });
   }
 };
 
 export class DuplicateResultFlyout extends React.Component<DuplicateResultFlyoutProps, State> {
   constructor(props: DuplicateResultFlyoutProps) {
     super(props);
-    this.state = { isLoading: false };
+    this.state = {
+      isLoading: false,
+      showRemainingButton:
```

Comment:
it looks more like a derived state computed from props, can we make it a `getter`?
```
get showRemainingButton() {
  return (
    props.successfulCopies.length > 0 &&
    !!props.failedCopies.find(
      ({ error }) => error.type === 'missing_references' || error.type === 'missing_data_source'
    ),
  );
}
```

Reply:
Sure I will update it